adsingestp/parsers/jats.py

```diff
@@ -913,22 +914,35 @@ def citation_context(
         self.back_meta = document.back
         body = document.body
         xrefs = body.find_all("xref")
+
+        # only include bibliographic references
```

Comment:
BeautifulSoup actually has a way to do a `find_all` and filter by both tag (e.g. `xref`) and attribute (`ref-type=bibr`) at the same time. There are a couple ways to write the syntax - I think either of these will work:
* `xrefs = body.find_all("xref", ref-type="bibr")`
* `xrefs = body.find_all("xref", attrs={"ref-type": "bibr"})`

The first one is a little shorter, the second one is a little clearer. Try one or the other and make sure it works. Assuming it does, you can edit line 916 and remove this for-loop altogether.

Reply:
The relevant section of the BS docs is here: https://www.crummy.com/software/BeautifulSoup/bs4/doc/#find-all

Reply:
Replaced the loop with the second option above.  Since the input test file only has `xrefs` with `ref-type="bibr"`, I ran this using my extraction code on random articles.  It is working as expected.